DOCUMENTATION.md

```diff
@@ -0,0 +1,101 @@
+# Documentation Guide
+
+M3 is a large and complex project, and any help you can offer to explain it better is most welcome.
+
+## Setup Documentation Locally
+
+### Prerequisites
+
+The M3 documentation uses [Hugo](https://gohugo.io/), a static site generator written in Go. [Find installation instructions in the Hugo documentation](https://gohugo.io/getting-started/installing/).
+
+If you also want to run and serve the M3 documentation, you need the custom theme, [Victor](https://github.com/chronosphereio/victor). Clone or download it into the same directory as the M3 codebase.
```

Comment:
Is Victor in the M3 repo? Because the Chronosphere repo is private pretty sure?

Reply:
@gibbscullen Nope, it's in the Chronosphere organisation, and that particular repo is public